ibis/backends/pyspark/compiler.py

```diff
@@ -14,18 +14,18 @@
 import ibis.expr.types as types
 from ibis import interval
 from ibis.backends.pandas.execution import execute
+from ibis.backends.pyspark.operations import PySparkTable
+from ibis.backends.pyspark.timecontext import (
+    combine_time_context,
+    filter_by_time_context,
+)
```

Comment:
Same here.

Reply:
Yup - updated.

Is the idea that moving to backends directory is a first step towards eventually having them in separate repos?

Reply:
That's an open question. Omnisci depends on the pymapd library, which created lots of trouble in conda (slow resolving, conflicts with other dependencies...). So, moving the omnisci backend out of this repo is already planned. For other backends we'll see. I personally think it's a good idea to move many of them to separate repos. Particularly if they have maintainers who know more about them than the Ibis core devs. But I think regardless on where they live, they should be properly decoupled and as standard as possible.